src/Functional.CQS.AOP.IoC.SimpleInjector.Tests/ContainerExtensionsTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using FluentAssertions;
+using Functional.CQS.AOP.CommonTestInfrastructure;
+using Functional.CQS.AOP.CommonTestInfrastructure.DummyObjects;
+using SimpleInjector;
+using Xunit;
+
+namespace Functional.CQS.AOP.IoC.SimpleInjector.Tests
+{
+	public class ContainerExtensionsTests
+	{
+		private static readonly Type[] _typesToCheck = new[]
```

Comment:
There are 6 handlers listed here, but 10 exist in the `DummyObjects` folder. Are the other 4 intentionally excluded here?

Reply:
They should probably be included for completeness sake, yes

Reply:
In a subsequent PR, I'll be moving these into their own `TestUtility` class so that multiple sets of tests can access them.  I'll add the other handler types at that time.